frontend/packages/core/src/WorkflowLayout/index.tsx

```diff
@@ -74,11 +81,11 @@ const WorkflowLayout = ({
   children,
 }: React.PropsWithChildren<LayoutProps>) => {
   const location = useLocation();
-  const workflowPaths = workflow.routes.map(({ path }) => `/${workflow.path}/${path}`);
+  const workflowPaths = workflow?.routes.map(({ path }) => `/${workflow?.path}/${path}`);
```

Comment:
Maybe this in case of no `workflow?.routes` so `workflowPaths` is not left `undefined`
```suggestion
  const workflowPaths = workflow?.routes.map(({ path }) => `/${workflow?.path}/${path}`) || [];
```